lighthouse-core/audits/unsized-images.js

```diff
@@ -40,15 +40,23 @@ class UnsizedImages extends Audit {
   }
 
   /**
-   * An img size attribute is valid for preventing CLS
-   * if it is a non-negative, non-zero integer.
-   * @param {string} attr
+   * An img size attribute is valid if it is a non-negative integer (incl zero!).
+   * @url https://html.spec.whatwg.org/multipage/embedded-content-other.html#dimension-attributes
+   * @param {string} attrValue
    * @return {boolean}
    */
-  static isValidAttr(attr) {
-    const NON_NEGATIVE_INT_REGEX = /^\d+$/;
-    const ZERO_REGEX = /^0+$/;
-    return NON_NEGATIVE_INT_REGEX.test(attr) && !ZERO_REGEX.test(attr);
+  static isValidAttr(attrValue) {
+    // First, superweird edge case of using the positive-sign. The the spec _sorta_ says it's valid...
+    // https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#rules-for-parsing-integers
+    //   > Otherwise, if the character is … (+): Advance position to the next character.
+    //   > (The "+" is ignored, but it is not conforming.)
+    // lol.  Chrome (at least) doesn't ignore but rejects this as a non-conforming value.
```

Comment:
```suggestion
    // lol.  Irrelevant though b/c Chrome (at least) doesn't ignore. It rejects this as a non-conforming value.
```

or any other phrasing that emphasizes a little more why we should ignore what the spec says :)

Reply:
good call thx.